Sources/GoogleAI/GenerativeAIRequest.swift

```diff
@@ -36,10 +36,9 @@ public struct RequestOptions {
   /// Initializes a request options object.
   ///
   /// - Parameters:
-  ///   - timeout The request’s timeout interval in seconds; if not specified uses the default value
-  ///   for a `URLRequest`.
-  ///   - apiVersion The API version to use in requests to the backend; defaults to "v1beta".
-  public init(timeout: TimeInterval? = nil, apiVersion: String = "v1beta") {
+  ///   - timeout: The request’s timeout interval in seconds; defaults to 300 seconds (5 minutes).
+  ///   - apiVersion: The API version to use in requests to the backend; defaults to "v1beta".
+  public init(timeout: TimeInterval? = 300.0, apiVersion: String = "v1beta") {
```

Comment:
It would technically be a breaking change but I wonder if we should make this a non-optional `TimeInterval` to avoid having 3 possible states (custom timeout when set, default of 300.0 and platform default when explicitly `nil`).

Reply:
What are the use cases for a separate default and platform default?

Reply:
I personally can't think of one beyond avoiding the breaking change (i.e., can no longer explicitly enter `nil`).

Reply:
I'm not sure it's a breaking change to change the default timeout interval - since the functionality is dependent on variable backend behavior.

Reply:
To clarify, the breaking change I'm referring to would be to change `TimeInterval? = 300.0` to `TimeInterval = 300.0`.

Another alternative would be to leave the public API as `TimeInterval? = nil`, change `let timeout: TimeInterval?` to `let timeout: TimeInterval` and then do `self.timeout = timeout ?? 300.0`.

Reply:
I get it now. Thanks. 

It would only break code that was explicitly passing nil to the optional. I would expect that to be rare and going to a non-optional with a default value, should be a fine break since we're still in preview.

Reply:
Agreed. I expect that would be exceedingly rare. Will update shortly.

Reply:
Updated to non-optional.